### DISCUSSION TITLE
DAPLink running in Ubuntu

### DISCUSSION THREAD

**Discussion by pengqiangChina** [ARMmbed/DAPLink#1049] [General]
Plug in DAPLink device, and SWD/JTAG function is OK, but I can't find any uart device(ttyUSB* or ttyACM* or ttyS*). It seems that there is no driver about usb to uart(WCH and FTDI chip are OK). Do anybody know how to use com function in Ubuntu?

**Comment by mbrossard** [comment]
What DAPLink device are you using? If it appears as storage device can you share the content of the `DETAILS.TXT` file?
